docs/stressors.md

```diff
@@ -0,0 +1,171 @@
+# Adding a new stressor to the application emulator
```

Comment:
Cool idea! I just see there is one thing missing perhaps, the logging functionality of customised stressors?

Reply:
Added!